lib/simulation/openfisca/mapping/reverse.js

```diff
@@ -4,10 +4,17 @@ var periods = require('./periods');
 var PRESTATIONS = require('../prestations');
 
 
-module.exports = function reverseMap(openFiscaFamille, date) {
+module.exports = function reverseMap(openFiscaFamille, date, injectedRessources) {
```

Comment:
I reorganized the two functions, giving to reverseMap the responsibility to compute the injected resources. Is it more clear ?